### PR TITLE
Improve Test Organization

### PR DESCRIPTION
**Pull request by Lakshmikanth2001**
- omitted the use of `exec` in all test cases
- Improve test grouping

### CODE REVIEW

**Comment by Lakshmikanth2001** [comment]
Any updates please

File: electricpy/__init__.py [hunk 1]
```diff
@@ -4656,17 +4656,26 @@ def syncspeed(Npol, freq=60, Hz=False):
                 Frequency of electrical system in Hertz, default=60
     Hz:         bool, optional
                 Boolean control to enable return in Hertz. default=False
+    rpm:        bool, optional
+                Boolean control to enable return in rpm. default
```
Comment (engineerjoe440) [inline electricpy/__init__.py:4660]: Please complete the `default=False` statement.

**Review by engineerjoe440 — CHANGES_REQUESTED** [review]
Just some little documentation stuff. Looks great! Thank you!

**Review by engineerjoe440 — APPROVED** [review]
Fantastic!!!! Thank you!

**Pull request merged**